### PR TITLE
Add eprime logits processor

### PR DESCRIPTION
**Pull request by deoxykev**
Implements a new logits processor that blocks forms of the verb "to be" 
(am, is, are, was, were, be, being, been) to enforce E-Prime writing style.
This promotes more precise language by avoiding essentialist statements and
encouraging active verbs instead of copula constructions.

The processor analyzes upcoming token probabilities and zeroes out logits
for banned words, allowing the model to find alternative phrasings that
avoid stating what things "are" in favor of describing relationships,
perceptions, and processes.

### CODE REVIEW

**Comment by aerdem4** [comment]
Thanks for your PR but your keyword list is larger than what is forbidden in e-prime as far as I understand. Also you don't have a notebook example for the users like any other processor in this repo.
